Dockerfile

```diff
@@ -1,38 +1,19 @@
-FROM ubuntu:14.04
-MAINTAINER Colin Alston <colin@praekelt.com>
-RUN apt-get update && apt-get -y --force-yes install libjpeg-dev zlib1g-dev libxslt1-dev libpq-dev nginx redis-server supervisor python-dev python-pip
-RUN apt-get -y install libffi-dev gettext
+FROM praekeltfoundation/django-bootstrap
+COPY . /app
```

Comment:
We want to do this step as late as possible so that changes to the source code don't invalidate all the cached image layers.